ClientApp/src/app/FlightForSearch.ts

```diff
@@ -1,6 +1,6 @@
 export interface FlightForSearch{
   departure?: number;
   arrival?: number;
-  date?: Date;
-  amount?: number;
+  date?: any;
+  // amount?: number;
```

Comment:
Consider removing this if you're not using it. Also what exactly is amount here for?